opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -104,6 +104,7 @@ namespace Opm {
         // units because internally to opm-parser everything is represented by SI
         // units...
         const UnitSystem& getDeckUnitSystem() const;
+        const UnitSystem& getUnits() const;
```

Comment:
I do not think we should have two functions doing the same; what about `getDeckUnits()` or `getInputUnits()`?